common/metrics/config.go

```diff
@@ -216,23 +216,26 @@ var (
 // customReporter Provide this argument if you want to report metrics to a custom metric platform, otherwise use nil.
 //
 // returns SeverReporter, SDKReporter, error
-func (c *Config) InitMetricReporters(logger log.Logger, customReporter interface{}) (Reporter, Reporter, error) {
-	// init from config only
-	if customReporter == nil {
-		if c.Prometheus != nil && len(c.Prometheus.Framework) > 0 {
-			return c.initReportersFromPrometheusConfig(logger, customReporter)
-		}
+func InitMetricReporters(logger log.Logger, c *Config, customReporter interface{}) (Reporter, Reporter, error) {
```

Comment:
Can we not expose this method? We don't want user to use this method.

Reply:
We have to expose it to be accessible by other server packages.